src/BenchmarkDotNet/Templates/WasmCsProj.txt

```diff
@@ -1,37 +1,33 @@
-﻿<Project Sdk="$SDKNAME$" DefaultTargets="$TARGET$">
+﻿<Project Sdk="$SDKNAME$" DefaultTargets="publish">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <OutputPath>bin</OutputPath>
     <RuntimeSrcDir>$RUNTIMESRCDIR$</RuntimeSrcDir>
+    <RuntimeConfig>Release</RuntimeConfig>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <TargetFramework>$TFM$</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <AppDir>$(MSBuildThisFileDirectory)\bin\$TFM$\browser-wasm\publish</AppDir>
     <AssemblyName>$PROGRAMNAME$</AssemblyName>
     <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
-    <WasmMainJSPath>$MAINJS$</WasmMainJSPath>
-    <MicrosoftNetCoreAppRuntimePackDir>$RUNTIMEPACK$</MicrosoftNetCoreAppRuntimePackDir>
-    <UsingBrowserRuntimeWorkload>false</UsingBrowserRuntimeWorkload>
+    <PublishTrimmed>false</PublishTrimmed>
+    <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
```

Comment:
Not sure whether we want to disable trimming above, is there an issue with linked assemblies?
 If so then `SuppressTrimAnalysisWarnings` property should not be needed as it is only used when trimming and running `illink`.

Reply:
I disabled trimming because it can potentially cause problems, and we only *really* need it on wasm-aot to keep aot compile times down. 

But I can get rid of the SupressTrimAnalysisWarnings.